ledger/store/trackerdb/sqlitedriver/sql.go

```diff
@@ -208,39 +208,6 @@ func MakeAccountsSQLWriter(e db.Executable, hasAccounts, hasResources, hasKvPair
 	return
 }
 
-// ListCreatables returns an array of CreatableLocator which have CreatableIndex smaller or equal to maxIdx and are of the provided CreatableType.
-func (qs *accountsDbQueries) ListCreatables(maxIdx basics.CreatableIndex, maxResults uint64, ctype basics.CreatableType) (results []basics.CreatableLocator, dbRound basics.Round, err error) {
-	err = db.Retry(func() error {
-		// Query for assets in range
-		rows, err := qs.listCreatablesStmt.Query(maxIdx, ctype, maxResults)
```

Comment:
nit: I guess this query `listCreatablesStmt` can be deleted too?

Reply:
yeah, I missed it. Just pushed.